packages/toolpad-app/src/toolpad/AppEditor/PageEditor/ComponentCatalog.tsx

```diff
@@ -15,44 +15,20 @@ import {
 } from '../../../toolpadComponents';
 
 interface FutureComponentSpec {
-  displayName: string;
-  githubLink: string;
+  displayName?: string;
+  url: string;
 }
 
-const FUTURE_COMPONENTS: FutureComponentSpec[] = [
-  {
-    displayName: 'Form',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/749',
-  },
-  {
-    displayName: 'Card',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/748',
-  },
-  {
-    displayName: 'Tabs',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/747',
-  },
-  {
-    displayName: 'Slider',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/746',
-  },
-  {
-    displayName: 'Switch',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/745',
-  },
-  {
-    displayName: 'RadioButton',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/744',
-  },
-  {
-    displayName: 'DatePicker',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/743',
-  },
-  {
-    displayName: 'Checkbox',
-    githubLink: 'https://github.com/mui/mui-toolpad/issues/742',
-  },
-];
+const FUTURE_COMPONENTS = new Map<string, FutureComponentSpec>([
+  ['Form', { url: 'https://github.com/mui/mui-toolpad/issues/749' }],
```

Comment:
Just curious - why would we omit `displayName` key and use array form instead? 🤔 Seems like even in the code below it could be bit simpler:
```
FUTURE_COMPONENTS = [
 { name, link }, ...
]
//...

Array.from(FUTURE_COMPONENTS, ({ name, link} => <Link key={name} href={link} />)
```

Reply:
because a map is a good code construct to communicate that these are intended to be unique, and a good runtime construct to guarantee it as well